### ISSUE TITLE
PR/Media/Outreach

### ISSUE THREAD

**Issue by roryfox** [EugTech/king-pong-plan-2019#26]
- [x] Create Press Release  (https://docs.google.com/document/d/1tpBNxzG06jabCN3t6Om40HR-H64fc-E2GN4rwkRfaww/edit?usp=sharing)
- [x] Submit PR to media list (Rory)
- [x] Get listed in Register Guard (Rory)
- [x] Get listed in Eugene Weekly (Rory)
- [x] Get listed in Lane Travel website (Rory/Jessica?)

**Comment by jessicabob** [comment]
I know the lady at travel lane county that does the website.

**Comment by roryfox** [comment]
Rocking!  Im gonna work on an updated Press Release later today and get team input before we start slamming it out into the community!  Ill double check we get listed on those event calendars as well.

**Comment by markdav-is** [comment]
looking good!  I encorperated @jessicabob comments and  added a shout-out to bitforest

**Comment by roryfox** [comment]
Thanks for the proofread and edits @jessicabob!  I plan to start getting this out there before wednesday, and sooner if possible as long as we are happy with it!

**Comment by jessicabob** [comment]
I went back over the formatting there were a few lines that were similar but different fonts. I think that it is ready to go, after the last empty page is deleted. Are you exporting as a pdf to attach to an email? I also think that the FB should get updated and the website should have a king pong thing on the top before we send it out. I am going to get the website part done today. @roryfox @nohorse

**Comment by roryfox** [comment]
@jessicabob The way the table was created in the OG document is causing the second page there...  I am just going to deal with it when I export it to PDF  so no big deal there.  Good eye on the fonts.  I had only noticed the one in the names section.  The FB update is a separate ticket assigned to mark.
